### ISSUE TITLE
ISMRM Oral Presentation

### ISSUE THREAD

**Issue by 62442katieb** [physiopy/phys2bids#392]
Hi @physiopy/all ! 

ISMRM is coming up and our abstract has been accepted for an oral scientific session! 🎉It's not due for a few weeks, but I figured we should get the discussion going here. The text of the email regarding this session and the process of preparing / presenting is copied below (with important points in bold):
>Program numbers have been assigned for oral presentations to be presented at the **ISMRM & SMRT Virtual Annual Meeting & Exhibition, to be held 15 - 20 May 2021**.
>
>The program number for your abstract, titled "Physiopy: A community-driven suite of tools for physiological recordings in neuroimaging," is:
>
>0726
>
> Please reference the above program number in all future correspondence with the ISMRM office regarding this abstract. ADDITIONALLY, PLEASE USE THIS NUMBER WHEN NAMING YOUR VIDEO FILE UPLOAD. PLEASE SEE INSTRUCTIONS LOCATED HERE: https://www.ismrm.org/21m/guidelines/.
> 
> This abstract will be featured in the following session:
> Parent Session Name: Data Processing & Software Tools
> Oral Session Name: Data Processing & Software Tools
> Session Day: **Thursday, 20 May 2021**
> Session Time: **14:00 UTC**
> 
> Our oral scientific sessions for the 2021 Annual Meeting will have a different format than in previous years. Like prior years, oral abstract sessions will consist of 10 abstracts. **We are asking our oral abstract presenters to upload their presentations, consisting of 5-minute talks, by 20 April 2021** (your upload instructions will follow closer to the submission date). Please do not wait to get started recording your videos! These presentations will be available for viewing by all attendees approximately two weeks before the Annual Meeting. During this time, attendees may post questions via a chat box associated with each presentation, and the presenter can answer these questions in the chat as well. Once the meeting opens, please log in frequently to check for questions and provide answers!
> 
> During the week of the Annual Meeting, **each oral abstract session will run for one hour** between the times of 12:00 UTC-20:00 UTC. Each oral abstract session will be managed by (at least) two moderators. In this hour, the **pre-recorded oral presentations will be played in a Zoom webinar format,** and attendees will be able to pose questions to the presenters in the chat; presenters, if available, may answer these questions also in the chat. Oral abstract presenters will not be able to answer questions verbally in this portion of the session.
> 
> **Following the hour-long oral session, small-group Zoom meetings will be held in which oral abstract authors and attendees can interact "face-to-face."** The presenter will be grouped with the presenters of 4 other abstracts from the same session, and attendees will be able to join and ask questions of any of these presenters; it will be possible for participants to share their screens and content in the chat box as well. We hope that these small groups will facilitate discussion about the scientific work.
> 
> We look forward to your participation in both the 1-hour webinar in which your abstract is featured and the following 1-hour Zoom meeting to describe your work and answer questions from the audience.

**TL;DR:** We / I need to prepare a 5-minute oral presentation on physiopy, record it, and submit by *April 20th*

I am planning to put together a few slides going through the content of the abstract (probably at the end of next week). Specifically:
1. Intro to the problem (lack of community, guidelines, tools for physio + fMRI in a research setting)
2. The physiopy community, best practices & recommendations
3. Phys2bids: encouraging sharing of physio data
4. Peakdet & phys2denoise: completing the physiopy ecosystem and enabling a reproducible workflow from collection to analysis
5. Future directions & hopes & please join us
## Outstanding questions

    - Is there anything in those 5 points above I should change / remove?
    - Is there anything missing in those 5 points above?
    - How would you like to be included in this process?

**Comment by eurunuela** [comment]
Thanks @62442katieb ! 

Given that it's a 5-minute talk, I wonder if we should emphasize one of the 5 points more than the others; i.e., what would be the most interesting for the ISMRM public? Would it make sense to emphasize on the need for the physiopy community to grow? Maybe we want to showcase the tools mostly?

What do you think?

**Comment by RayStick** [comment]
Thanks @62442katieb - great that you've got this started. Happy to review slides/presentations, as needed! 
I agree with @eurunuela that with a 5 minute talk you may want to focus in on a few points, and only briefly mention others. Would there be enough time for a software demo in 5 mins? If not, I guess more of these points can be covered

**Comment by 62442katieb** [comment]
The video has been submitted! 🎉
